src/bun.js/webcore/body.zig

```diff
@@ -308,6 +292,36 @@ pub const Body = struct {
         Error: JSValue,
         Null: void,
 
+        pub fn initRef(value: Value) *BodyValueRef {
```

Comment:
Don't we only need this for requests which have bodies instead of allocating one of these for requests which do not have bodies?